Components/templateWeather.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
 
 import {Text, View, StyleSheet} from 'react-native';
-import {Forecast} from './weather.type';
+import {ForecastData} from './weatherData.type';
 
 const KELVIN_OFFSET: number = 273;
 const METER_PER_SECOND_RATIO: number = 3.6;
 
 interface Props {
-  forecastElement: Forecast;
+  forecastElement: ForecastData;
 }
 
 export class TemplateWeather extends React.Component<Props> {
   render() {
-    const forecast: Forecast = this.props.forecastElement;
+    const forecast: ForecastData = this.props.forecastElement;
 
     const convertKelvinToCelcius = (temperature: number) =>
```

Comment:
Actually, does this need to be in the component ? 
It should be in a small service (that can be tested ;) ;) )